### PR TITLE
Draft content styleguide

### PR DESCRIPTION
**Pull request by sknep**
## Changes proposed in this pull request:
Adds much more detail and standardizes some voice and tone

## security considerations
n/a unless we want to add some security language

### CODE REVIEW

File: ContentGuide.md [hunk 1]
```diff
@@ -1,19 +1,188 @@
...
-## Use these exact strings for tricky product names, abbreviations, and team names
+-   **Assume everyone is drowning in alphabet soup**: Always use full names on first reference, before using acronyms, except for messages to other TTS audiences. For now, it's usually less relevant to mention that we're within Solutions or FAS.
+    >   "Cloud.gov, part of the Technology Transformation Services (TTS) within GSA, supports federal agencies build secure, resilient web applications."
+
+-   **Be transparent and collaborative**: Emphasize teamwork and shared goals, and share the limelight. Be a good neighbor.
+    >   "We're proud of our colleages at 18F who worked on this prototype, which was powered by Cloud.gov."
+
+### 2\. Referring to external or non-government entities
+
+-   **Always visually distinguish external links** and compy with [GSA's external linking policy](https://www.gsa.gov/website-information/website-policies#:~:text=We%20provide%20these%20links%20as,included%20in%20these%20website%20links.)
```
Comment (bengerman13) [inline ContentGuide.md:166]:
```suggestion
-   **Always visually distinguish external links** and comply with [GSA's external linking policy](https://www.gsa.gov/website-information/website-policies#:~:text=We%20provide%20these%20links%20as,included%20in%20these%20website%20links.)
```

File: ContentGuide.md [hunk 2]
```diff
@@ -1,19 +1,188 @@
...
 
-**You should add things too:** Just make a pull request.
+## How to talk about everyone else
 
-For additional helpful background knowledge about writing for cloud.gov users, see the [18F Content Guide](https://pages.18f.gov/content-guide/), including its [content principles](https://pages.18f.gov/content-guide/content-principles/).
+### 1\. Referring to other offices, programs, or teams within TTS or GSA
 
-## Use these exact strings for tricky product names, abbreviations, and team names
+-   **Assume everyone is drowning in alphabet soup**: Always use full names on first reference, before using acronyms, except for messages to other TTS audiences. For now, it's usually less relevant to mention that we're within Solutions or FAS.
+    >   "Cloud.gov, part of the Technology Transformation Services (TTS) within GSA, supports federal agencies build secure, resilient web applications."
```
Comment (bengerman13) [inline ContentGuide.md:159]: Should `GSA` be `the U.S. General Services Administration (GSA)`? or should it be called out as an exception to the rule above?

File: ContentGuide.md [hunk 3]
```diff
@@ -88,44 +255,18 @@ These words should be lowercased in the middle of a sentence and capitalized at
...
-
-Use active voice, especially when you're writing technical explanations. For example: *"the subject did something"*, such as *"the cluster recorded the data"*.
-
-If you write sentences in passive voice (*"something was done"*), such as *"the data was recorded"*, your sentence is hiding important technical information: what did the action? What recorded the data? If you write *something was done by the thing*, revise it to say *the thing did something*.
-
-Using active voice is important for outage reports and postmortems. If we write with passive voice, this can sound like we're not taking full responsibility for our actions and mistakes.
-
-Active voice is also important for documentation (including compliance documentation). When you state which part of the system is doing which action, this helps readers understand how the system works.
-
-## Use contractions and friendly language
```
Comment (pburkholder) [inline ContentGuide.md:101]: Why did contractions, abbreviations, and statements v. questions get the axe?

**Review by kfoley-18F — APPROVED** [review]
YESSSSS!!

Thank you for codifying and writing this down!
This is a fabulous update, and will help to bring others into writing consistently (on cg-site, when emailing customers, in preso decks) with a "Cloud.gov" voice.

**Pull request merged**